packages/components/src/components/status-badge/status-badge.scss

```diff
@@ -14,10 +14,9 @@
         &--pending {
             color: var(--status-warning-dark);
             background-color: var(--status-warning-transparent);
-            text-decoration: underline;
```

Comment:
we need this styling in tradershub dashboard. i will fix it in my PR for now. will merge this now